skimage/filters/_frangi.py

```diff
@@ -0,0 +1,186 @@
+u"""Implemnetation of Frangi and Hessian filters support.
```

Comment:
There is a non-ASCII character somewhere, so pydocstyle suggested adding unicode docstring instead of regular one.